src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -71,7 +74,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $io->section('Variables');
         $io->table(
-            array_merge(['Variable', 'Value'], $availableFiles),
+            array_merge(['Variable', 'Value'], array_map(static function (string $availableFile) {
+                return basename($availableFile);
+            }, $availableFiles)),
```

Comment:
We show full path of scanned Files above.
I think only name is enough else table might be too wide

Without this change
```
\n
 ---------- ------- ------------------------------- --------------------------- \n
  Variable   Value   sub/path/custom.env.local.php   sub/path/custom.env.local  \n
 ---------- ------- ------------------------------- --------------------------- \n
  FOO        BaR     BaR                             n/a                        \n
  TEST       1234    1234                            1111                       \n
 ---------- ------- ------------------------------- --------------------------- \n
\n
```
With this change
```
\n
 ---------- ------- ---------------------- ------------------ \n
  Variable   Value   custom.env.local.php   custom.env.local  \n
 ---------- ------- ---------------------- ------------------ \n
  FOO        BaR     BaR                    n/a               \n
  TEST       1234    1234                   1111              \n
 ---------- ------- ---------------------- ------------------ \n
\n
```

Reply:
You are right. But to be clear to the user, the configured path with directory should be displayed before the table, when there is a directory. So that they know where are located the files.

Reply:
```suggestion
            array_merge(['Variable', 'Value'], array_map('basename', $availableFiles)),
```